File: Java/set2/Q5.java
// 5. Write a program to find biggest of three numbers by using logical operator by taking input from the user. 

class Q5
{
    public static void main(String[]args)
    {
        int a=6, b=3, c=0;
        if(a>b && a>c)
            System.out.println(a+" is bigger");
        else if(b>a && b>c)
                System.out.println(b+" is bigger");
            else
                System.out.println(c+ " is bigger");
        }
}

File: Python/set2/Q5.py
# 5. Write a program to find the distance between the two points in Cartesian coordinate system

import math
x1=float(input("Enter co-ordinates of x1"))
y1=float(input("Enter co-ordinates of y1"))
x2=float(input("Enter co-ordinates of x2"))
y2=float(input("Enter co-ordinates of y2"))

dist=math.sqrt((math.pow((x1-x2),2))+(math.pow((y1-y2),2)))
print("Distance:",dist)